cluster-service/Makefile

```diff
@@ -80,8 +80,10 @@ deploy:
 	  --set pullBinding.workloadIdentityTenantId="$${IMAGE_PULLER_MI_TENANT_ID}" \
 	  --set pullBinding.registry=${ACR_NAME}.azurecr.io \
 	  --set pullBinding.scope=repository:${IMAGE_REPO}:pull \
+		--set tracing.address=${TRACING_ADDRESS} \
```

Comment:
The indentation looks odd. Not sure if this GH UI issue or this is an actual spacing issue.